lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -44,7 +44,11 @@ class RedisCache extends CacheProvider
      */
     public function setRedis(Redis $redis)
     {
-        $redis->setOption(Redis::OPT_SERIALIZER, $this->getSerializerValue());
+        // The redis object is injected from an external service
+        // Serializer option should be set outside
+        // We should not force the serializer option here
+        // This code is left here for reference only
```

Comment:
code should not be left for reference. It should be removed if it is not used anymore (I cannot tell whether the doctrine/cache maintainers will accept it or no though)